src/server/models/job.ts

```diff
@@ -59,12 +71,15 @@ export const JobItem = <JobItemStatic>sequelize.define(
     message: {
       type: Sequelize.STRING,
       defaultValue: '',
-      allowNull: false,
     },
     params: {
       type: Sequelize.JSON,
       allowNull: false,
     },
+    jobItemId: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
```

Comment:
Spoke about a similar issue previously with Dat (whether we could use uuid as primary key instead) - he was slightly hesitant because we wanted to keep the primary key consistent with the rest of the tables we have in the db (primary key on auto-incrementing id).

We could monitor for performance issues. If this becomes an issue, we can add an index on `jobItemId` as opposed to making it the primary key?

Reply:
i see! sounds good to leave this as a non-pkey then. would adding a unique constraint here instead make sense?

Reply:
Sure I think that could make sense!

Reply:
[fd83ace](https://github.com/opengovsg/GoGovSG/pull/2085/commits/fd83ace7a1cfa25decc7a0fdc0a7c16c0a9a270c)